js/game.js

```diff
@@ -0,0 +1,46 @@
+/* ----------------------- Element Variables -----------------------*/
+const slots = document.getElementsByClassName('circle');
+const resetBtn = document.getElementById('reset');
+const playerBtns = document.getElementsByClassName('player-btn');
+
+/* ----------------------- Other Variables -----------------------*/
+
+let board = new Board(slots, playerBtns);
+
+const numColumns = 7;
+const numSlots = 42;
+
+/* ----------------------- Event Handling -----------------------*/
+
+resetBtn.addEventListener('click', () => board.reset());
+
+for (let i = 0; i < board.length(); i++) {
+    const slot = board.getSlot(i);
+
+    // find index for last slot in current column
+    let lastSlotNum = i;
+    while ((lastSlotNum + numColumns) < numSlots) {
```

Comment:
there might be a more elegant way to do this; think about what you'll need to determine whether the game is finished.